lib/tesla_api/vehicle/state.ex

```diff
@@ -362,6 +364,7 @@ defmodule TeslaApi.Vehicle.State do
       :exterior_color,
       :has_air_suspension,
       :has_ludicrous_mode,
+      :key_version,
```

Comment:
Are these new?

Reply:
I found them in the [unofficial API documentation](https://tesla-api.timdorr.com/vehicle/state/vehicleconfig). This attribute in particular was added for the sentry mode: https://github.com/timdorr/tesla-api/commit/afe8bf83f9d64f0ac612708d3d71a9d634755552#diff-e9a7827b47ba8ad6e153bb6b4666826bR21